### PR TITLE
sail 0.9.10

### PR DESCRIPTION
**Pull request by dtrodrigues**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
==> /opt/homebrew/Cellar/sail/0.9.10/bin/sail --version
Error: sail: failed
An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - /opt/homebrew/Cellar/sail/0.9.10/bin/sail
```

File: Formula/sail.rb [hunk 1]
```diff
@@ -47,6 +47,11 @@ class Sail < Formula
     sha256 "b8aa58f8cf793ffd8782d3d8cb19e66ef36f7aba4353eec859e74678b01b07a7"
   end
 
+  resource "sailed.io" do
+    url "https://files.pythonhosted.org/packages/e2/1c/9c2dc6d5ce5ca8017f1d40271999c7f11e44cbcbd8a6fb68b965284a6dd3/sailed.io-0.9.10.tar.gz"
+    sha256 "d54a504ca6aa8c175727507737ef0c8ceeee1e38cfc5742bc1561b158f13f1ba"
+  end
```
Comment (cho-m) [inline Formula/sail.rb:53]: How did the source tarball get added as a resource via `brew bump-formula-pr`?

Comment (chenrui333) [reply Formula/sail.rb:53]: the formula is named `sail`, but the resource is named `sailed.io` 🤦

File: pypi_formula_mappings.json [hunk 2]
```diff
@@ -203,6 +203,7 @@
   "s4cmd": {
     "exclude_packages": ["six"]
   },
+  "sail": "sailed-io",
```
Comment (chenrui333) [inline pypi_formula_mappings.json:206]: 👍

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**